api/soap/mantisconnect.wsdl

```diff
@@ -45,6 +45,7 @@
    <xsd:element name="account_data" type="tns:AccountData" minOccurs="0"/>
    <xsd:element name="access_level" type="xsd:integer" minOccurs="0"/>
    <xsd:element name="timezone" type="xsd:string" minOccurs="0"/>
+   <xsd:element name="api_token_type" type="xsd:integer" minOccurs="0"/>
```

Comment:
Do we need to return such information to enable bug reporting scenario?  I would rather minimize the scope of this change to what is required to light up the scenario.